cypress/e2e/inventory/holdings/verify-holdings- source-added-holdings-manually.cy.js

```diff
@@ -0,0 +1,66 @@
+import { including } from '@interactors/html';
+import { randomFourDigitNumber } from '../../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../../support/dictionary';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import Users from '../../../support/fragments/users/users';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryNewHoldings from '../../../support/fragments/inventory/inventoryNewHoldings';
+
+const testData = {
+  user: {},
+};
+const item = {
+  instanceName: `Inventory Instance ${randomFourDigitNumber()}`,
+  itemBarcode: randomFourDigitNumber(),
+};
+const permanentLocation = 'Annex (KU/CC/DI/A)';
+const calloutMessage = 'has been successfully saved.';
+
+describe('Holdings', () => {
+  before('Create test data', () => {
+    cy.createTempUser([
+      Permissions.inventoryAll.gui,
+      Permissions.uiInventoryViewInstances.gui,
+      Permissions.uiInventoryViewCreateEditHoldings,
+      Permissions.uiInventoryViewCreateEditItems,
```

Comment:
permissions don't match with the test case

Reply:
Many methods like InventoryInstance.waitLoading(); depends on elements that doesn't exists with View Permission only (from test case)

Reply:
then don't use those methods

Reply:
done